src/macaron/slsa_analyzer/analyzer.py

```diff
@@ -324,7 +324,7 @@ def run_single(
                 status=SCMStatus.ANALYSIS_FAILED,
             )
 
-        if not provenance_payload and parsed_purl and not config.get_value("path"):
+        if not provenance_payload and parsed_purl:
```

Comment:
I am a bit unclear on why we remove `not config.get_value("path")` here.

Reply:
I think previously we wanted to prevent provenance discovery for runs where a repo path has been provided. In the current state, I could not see a good reason to keep it, while removing it allows for more provenance discovery in cases where there is otherwise none.